dat/missions/zalek/neburesearch/neburesearch_00.lua

```diff
@@ -0,0 +1,221 @@
+text[3] = _([["You do not have enough free cargo space to accept this mission!"]])
+text[4] = _([["Great! I'll start to load the sensors into your ship right away. We should be ready to take off soon."]])
+text[5] = _([["So it is not a problem at all? I'm still a student and spent all funds I got on the sensor suit. Thank you for helping me out here! I'll start to load the sensors into your ship right away. We should be ready to take off soon."]])
+text[6] = _([[As you enter your ship you notice dozens of cables of various colors stretched across your ship's corridors. It is a complete mess. Even weirder is the fact that it is less than one period ago that you met the student in the bar. What has he done to your ship?
+    You follow the direction where most of the cable seem to lead to. Finally you find the culprit.]])
+text[7] = _([["Oh, hello again, Captain! I'm done just by now here, so we can take off whenever you're ready. I have to calibrate the sensors during the flight so there is no need to rush. Our first destination is %s."
+    After asking what he has done to your ship he just answers, "Why, I just installed the sensors as appointed. It should have no unwanted side effects on your ship."]])
+text[8] = _([["Just a complete mess? Haven't you noticed the color coding? Well, I have my own system so it may appears confusing to you. Don't worry, I know exactly what I'm doing!"
+    His last words are supposed to be reassuring but instead you realize accepting this mission was not the best idea...]])
+text[9] = _([[After reaching the %s system the student enters your cockpit. As he realized he forgot to even tell you his name he introduces himself as Robert Hofer and claims to be a student of professor Voges himself. You haven't ever heard of that name but it sounds like he is famous within House Za'lek.
+    "I will now start with the measurements. The density of the nebula is lower in this sector so that it is less volatile. For the real measurements we have to enter %s. I will tell you once we're done here."]])
+text[10] = _([[Suddenly you lose control over your ship. Apparently most core systems were shut down. Something drains your ship's energy and there are black outs in several parts of your ship.
+    You realize that your shields are down as well. In an environment like this... That's it, your going to die here! You knew accepting this mission was a mistake from the very first moment.]])
+text[11] = _([[It seems you are still alive and all core components are back online!
+    There is a certain someone you should have a serious talk to.]])
+text[12] = _([["Sorry for causing trouble. I'm not quite familiar with the electronics of this ship type. You really should fly a Za'lek ship instead. It wouldn't have possibly happened!"
```

Comment:
Worth noting that the Za'lek ships are military designs, so I don't think the Za'lek would expect civilians to fly those (and in fact you can't without already being a Za'lek ally).

🕵️

Reply:
I'm not sure if the student actually knows that. The point here is a lame excuse anyway.

Reply:
Ah, naïve upper-class person? Sounds legit.

🕵️